model/database/metrics.yaml

```diff
@@ -133,3 +134,21 @@ groups:
     attributes:
       - ref: db.client.connection.pool.name
         requirement_level: required
+
+  - id: metric.db.query.limit
+    type: metric
+    metric_name: db.query.limit
```

Comment:
BTW what would be the use-case for this metric? Why it's important to know the max number? What would users do with this information?

Reply:
I agree, this metric might not be very important because actual item count is already captured

Reply:
removing this metrics as of now